doc/radosgw/notifications.rst

```diff
@@ -394,31 +394,33 @@ The response has the following format:
         </ResponseMetadata>
     </GetTopicAttributesResponse>
 
-- User: the name of the user that created the topic.
-- Name: the name of the topic.
-- EndPoint: The JSON-formatted endpoint parameters, including:
-   - EndpointAddress: The push-endpoint URL.
-   - EndpointArgs: The push-endpoint args.
-   - EndpointTopic: The topic name to be sent to the endpoint (can be different
+- ``User``: The name of the user that created the topic.
```

Comment:
I would not end these with periods.  My rule of thumb is that if a list item is a sentence, it gets a period, otherwise it doesn't.  This is minor.

Reply:
Thank you.  
Could you please elaborate if you'd then also begin them with a lower case letter? I could then check the various parameter lists in the whole file.

Reply:
When the text is a literal, like ``user`` my sense is to leave it upper or lower as it actually appears live. '

Reply:
Sure, but what about how to begin the description of the parameter? For example here should it be

```suggestion
- ``User``: The name of the user that created the topic
```

Or
```suggestion
- ``User``: the name of the user that created the topic
```

Reply:
I'll push an update soon.